tests/test_response/test_room_response.py

```diff
@@ -76,82 +66,83 @@ def test_filter_by_number_of_invalid_responses(self):
 
     def test_search_response_room_name_only(self):
         """
         Testing for searching the room responses
         by room name
         """
         CommonTestCases.admin_token_assert_equal(
-            self,
-            search_response_by_room_only,
-            filter_by_response_data
-        )
+            self, search_response_by_room_only, filter_by_response_data)
 
     def test_filter_search_response_invalid_room(self):
         """
         Test for filter by response and search
         the room with invalid room name
         """
         CommonTestCases.admin_token_assert_in(
-            self,
-            search_response_by_room_invalid_room_query,
-            "No response for this room, enter a valid room name"
-        )
+            self, search_response_by_room_invalid_room_query,
+            "No response for this room, enter a valid room name")
 
     def test_search_response_invalid_room(self):
         """
         Test for search by room name
         with invalid room name
         """
         CommonTestCases.admin_token_assert_in(
-            self,
-            search_response_by_invalid_room,
-            "No response for this room, enter a valid room name"
-        )
+            self, search_response_by_invalid_room,
+            "No response for this room, enter a valid room name")
 
     def test_search_response_beyond_limits(self):
         """
         Test for search by room name
         beyond upper and lower limits
         """
         CommonTestCases.admin_token_assert_in(
-            self,
-            search_response_by_room_beyond_limits_query,
-            "No response for this room at this range"
-        )
+            self, search_response_by_room_beyond_limits_query,
+            "No response for this room at this range")
 
     def test_get_paginated_responses(self):
         """
         Test for paginated responses
         """
         CommonTestCases.admin_token_assert_equal(
-            self,
-            query_paginated_responses,
-            query_paginated_responses_response
-        )
+            self, query_paginated_responses,
+            query_paginated_responses_response)
 
     def test_get_paginated_responses_empty_page(self):
         """
         Test for paginated responses for empty page
         """
+        CommonTestCases.admin_token_assert_in(
+            self, query_paginated_responses_empty_page, "Page does not exist")
+
+    def test_mark_response_as_resolved_with_an_invalid_id(self):
+        """
+            Tests an admin can not mark
+            a response that does not exist
+            in the db as resolve
+        """
         CommonTestCases.admin_token_assert_in(
             self,
-            query_paginated_responses_empty_page,
-            "Page does not exist"
-        )
+            mark_response_as_resolved_mutation_with_an_invalid_response_id,
+            "Response does not exist")
+
+    def test_mark_response_as_resolved(self):
```

Comment:
Can admin mark resolved response unresolved?

Reply:
Yes that he can. But this test was for marking resolved because the response saved in the test database has a resolved field as False.

Reply:
I have added a test fo marking a response as unresolved